src/WebJobs.Script/Description/NodeFunctionInvoker.cs

```diff
@@ -280,12 +283,8 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
             Type triggerParameterType = input.GetType();
             if (triggerParameterType == typeof(string))
```

Comment:
I know that was there before, but why do type equality instead if `is` check on the `input` object?